src/components/error-fallback/error-fallback.tsx

```diff
@@ -0,0 +1,18 @@
+import { FC } from "react";
+
+interface Props {
+  error: Error;
+  onErrorReset?: () => void;
+};
+
+export const ErrorFallback: FC<Props> = ({ error, onErrorReset }) => {
+  return (
+    <section role="alert">
+      <h2>Something went wrong:</h2>
+
+      <pre>{error.message}</pre>
+
+      <button onClick={onErrorReset}></button>
```

Comment:
add resetButtonText prop, but the better approach would be if you move the button out of the component and pass it as children

Reply:
resolved